src/utils/format/formatCurrency.ts

```diff
@@ -59,7 +60,7 @@ export class CurrencyUtils {
       return parseWad(this.weiToUsd(amount)?.toString())
     if (targetCurrency === 'ETH' && this.usdPerEth !== undefined)
       return BigNumber.from(amount)
-        .div(this.usdPerEth * 100)
+        .div(round(this.usdPerEth * 100))
```

Comment:
Turns out I don't think we've ever actually converted USD->ETH in the app. There was a bug here that `this.usdPerEth * 100` could be a decimal, which breaks the site

Reply:
classic. nice catch